app/main/posts/views/post-view-data.directive.js

```diff
@@ -456,7 +457,7 @@ function PostViewDataController(
     function resetPosts() {
         $scope.posts = [];
         $scope.groupedPosts = {};
-        $scope.totalItems = $scope.itemsPerPage;
+        $scope.totalItems = 0;
```

Comment:
@Angamanga do you remember if there's a good reason why we had itemsPerPage (ie what might break with this fix? ) 
It fixes the issue of `0 of 20 posts` being shown when there are 0 posts (it shows 0 of 0 , instead), and it also makes it possible to block the bulk actions when there's zero posts.. but I'd appreciate if you took a look and let us know.

Reply:
I have really tried to break it but I can't. I suspect it is remains of all the refactors we did on that page when building it? Otherwise it looks good, but found 2 other bugs (not related to this change), will file them separately.